src/lib/core/utils/data-element-constraints.ts

```diff
@@ -170,6 +178,12 @@ export function mergeConstraints(
     keys.map((key): [string, DataElementConstraint] => {
       const constraintA = constraintMapA[key];
       const constraintB = constraintMapB[key];
+      const mergedIsTemporal =
```

Comment:
This looks like a no-op.  It's not used below.

Do you mean to return
```
  isTemporal: mergedIsTemporal
```
in the return object?

Reply:
yes. thanks.